src/lib/utils.test.ts

```diff
@@ -224,19 +286,123 @@ describe('utils', () => {
         encodedValue:
           '0x002000000003ca41e4ea94c8fa99889c8ea2c8948768cbaf4bc03e89ad98ffffffff002000000002f70ce3b58f275a4c28d06c98615760dde774de57ffffffff760d9bba002000000001d3236aa1b8a4dde5ea375fd1f2fb5c354e686c9fffffffffffffffff',
       },
+      {
+        schema: {
+          name: 'TupleWithArray',
+          key: '0x4b80742de2bf393a64c70000<address>',
+          keyType: 'MappingWithGrouping',
+          valueType: '(bytes4,bytes8,bytes32[CompactBytesArray])',
```

Comment:
Shouldn't this be `uint64`? Since the `valueContent` is `Number`

```suggestion
          valueType: '(bytes4,uint64,bytes32[CompactBytesArray])',
```